### ISSUE TITLE
Can Allele specific mapping handle polyploids?

### ISSUE THREAD

**Issue by Adamtaranto** [nservant/HiC-Pro#598]
Is it possible to make an allele-specific map with for genomes with ploidy > 2?

**Comment by nservant** [comment]
I do not think that the ploidy is an issue. However, it can only handle two alleles so far, is that your point ?

**Comment by ifoo1213** [comment]
I am guessing this question is about how to remove the multiple mapping pairs and only keep the unique mapping reads?
